src/mmg3d/mmg3d1.c

```diff
@@ -1134,13 +1166,19 @@ static int MMG5_coltet(MMG5_pMesh mesh,MMG5_pSol met,int8_t typchk) {
 
         if ( ilist > 0 ) {
           ier = MMG5_colver(mesh,met,list,ilist,iq,typchk);
-          if ( ier < 0 ) return -1;
+          if ( ier < 0 ){
+            MMG5_show_tet_location(mesh, pt, k);
```

Comment:
Please, keep this in debug mode only (`-d` command line option, `mesh->info.ddebug` variable inside the code) : 
```
if ( mesh->info.ddebug ) {
   MMG5_show_tet_location(mesh, pt, k);
}
```